dojo/tools/vulners/api_client.py

```diff
@@ -0,0 +1,78 @@
+import requests
```

Comment:
It seems that you implemented from scratch the client, any reason to not use the Pypi official package https://pypi.org/project/vulners/#description ?

Reply:
Just to do not include one more library to project

Reply:
it's better to use this package honestly. somebody will need to maintain this part. it's better to let vulners do it itself (this package is promoted through their doc), also the package is small (in kb)

Reply:
Right now vulners SDK does not support using of another API address instead of https://vulners.com
Few days ago was release of https://github.com/vulnersCom/vulners-proxy, so I can tell DD to use proxy instead of main API
As soon as SDK support proxy, I'll change raw requests to vulners SDK

Reply:
Made a review of the official SDK and it support proxies.

https://github.com/vulnersCom/api/blob/f56dab4ffcdb400da58b680d7feb89cae3550984/vulners/api.py#L97-L105

![image](https://user-images.githubusercontent.com/1694940/187021399-fe2ad658-6df6-4d10-9909-0380305c8a8a.png)

Reply:
@damiencarol, issues fixed. Will you have a chance to check it out?

Reply:
taking a look.